spec/spec_helper.rb

```diff
@@ -29,6 +28,8 @@
 require_relative "support/hint_examples.rb"
 require_relative "support/transformation_examples.rb"
 
+require 'alchemy/test_support/essence_shared_examples'
```

Comment:
Ah, sorry for not replying earlier. Line 27 would've worked as well. As you have noticed, just having `rspec` loaded is the important bit.